packages/uiweb/src/lib/dataProviders/ChatDataProvider.tsx

```diff
@@ -0,0 +1,29 @@
+import React, {useState} from "react";
+import { ENV } from "../config";
+import { ChatDataContext, IChatDataContextValues } from "../context/chatContext";
+
+export interface IChatUIProviderProps {
+    children: React.ReactNode;
```

Comment:
We also need account , env , prgpPrivateKey as input here. Also keep theme but keep it commented.